### PR TITLE
Standard library ptr routines cleanup

### PR DESCRIPTION
**Pull request by charlieMonroe**
Updating proposal to include requested change by the core team.

### CODE REVIEW

File: proposals/0127-cleaning-up-stdlib-ptr-buffer.md [hunk 1]
```diff
@@ -57,13 +57,11 @@ withUnsafePointer(to: &x) { (ptrX) -> Void in
 
 ---
 
-`unsafeAddressOf` is removed, in favor of adding a `unsafeAddress` field on `ObjectIdentifier`.
-`ObjectIdentifier` already contains a raw pointer in the internal `_value` field and 
-can be initialized with `AnyObject` just like the argument of `unsafeAddressOf`.
+`unsafeAddressOf` is removed. Suggested replacement is to use `Unamanged.passUnrepassUnretained`.
```
Comment (benrimmington) [inline proposals/0127-cleaning-up-stdlib-ptr-buffer.md]: Typo: `Unamanged.passUnrepassUnretained`

Comment (charlieMonroe) [inline proposals/0127-cleaning-up-stdlib-ptr-buffer.md]: Oops. Thanks for noticing. Fixing it now.

File: proposals/0127-cleaning-up-stdlib-ptr-buffer.md [hunk 2]
```diff
@@ -57,13 +57,11 @@ withUnsafePointer(to: &x) { (ptrX) -> Void in
 
 ---
 
-`unsafeAddressOf` is removed, in favor of adding a `unsafeAddress` field on `ObjectIdentifier`.
-`ObjectIdentifier` already contains a raw pointer in the internal `_value` field and 
-can be initialized with `AnyObject` just like the argument of `unsafeAddressOf`.
+`unsafeAddressOf` is removed. Suggested replacement is to use `Unamanged.passUnretained`.
```
Comment (xwu) [inline proposals/0127-cleaning-up-stdlib-ptr-buffer.md]: Typo here (`Unmanaged`)

Comment (charlieMonroe) [inline proposals/0127-cleaning-up-stdlib-ptr-buffer.md]: Thanks for noticing! Fixed.

**Comment by dabrahams** [comment]
@atrick Should we accept these changes?

**Comment by atrick** [comment]
Yes. It looks like this should have been merged long ago.

**Pull request merged**